fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroupMgr.java

```diff
@@ -137,6 +137,18 @@ public void resetQueryQueueProp() {
     }
 
     public WorkloadGroupMgr() {
+    }
+
+    public static WorkloadGroupMgr read(DataInput in) throws IOException {
+        String json = Text.readString(in);
+        return GsonUtils.GSON.fromJson(json, WorkloadGroupMgr.class);
```

Comment:
Add check if the default workload group is inited here and put it into workload group manager if not exist.